addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -39,9 +39,11 @@
         <field name="model">mailing.list</field>
         <field name="arch" type="xml">
             <form string="Contact List">
-                <header>
+                <header invisible="not id">
```

Comment:
Why ? With auto-save + action that anyway automatically saves, why doing that ? We would have to do it in all odoo headers ... which we don't do , as it is mainly noise in my opinion.

Reply:
Yes good point, we can stay with auto save I think.